app/views/organisations/students/show.html.erb

```diff
@@ -13,9 +13,26 @@
     </div>
   </section>
   <section class="bg-white">
-    <div class="max-w-5xl mx-auto p-6">
+    <div class="max-w-5xl mx-auto p-5">
```

Comment:
@vinutv 

This one as well for mobiles and tablets?

![image](https://github.com/pupilfirst/pupilfirst/assets/53794102/40ddbba7-d90a-4aed-bb2f-77c5a6ceb74e)